pkg/kv/kvserver/batcheval/cmd_scan_interleaved_intents.go

```diff
@@ -20,10 +20,20 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/storage"
 	"github.com/cockroachdb/cockroach/pkg/storage/enginepb"
 	"github.com/cockroachdb/cockroach/pkg/util/protoutil"
+	"github.com/cockroachdb/cockroach/pkg/util/syncutil"
 	"github.com/cockroachdb/errors"
+	"github.com/marusama/semaphore"
```

Comment:
I think we like this thing better (that sem isn't fair) https://github.com/cockroachdb/cockroach/blob/master/pkg/util/limit/limiter.go#L47